docs/developer/payments/stored-payments.mdx

```diff
@@ -11,19 +11,6 @@ Saleor uses synchronous webhooks to notify the Payment App about actions related
 
 :::info
 
-This feature was introduced in **Saleor 3.15**.
-
-:::
-
-:::caution
-
-This feature is in the **Feature Preview** stage, which means that it is available for experimentation and
```

Comment:
Same concerns as for checkout.problems.

The interface was delivered, but I would get the feedback first from app developers who were using the storedPaymentMethod API, before marking this as stable.